src/ai/backend/storage/api/manager.py

```diff
@@ -350,6 +353,7 @@ class Params(TypedDict):
                 {
                     t.Key("volume"): t.String(),
                     t.Key("vfid"): tx.VFolderID(),
+                    t.Key("mode", default=None): t.Null | t.Int,
```

Comment:
Is it not possible to set the value of `DEFAULT_VFOLDER_PERMISSION_MODE` as a default argument?